### PR TITLE
Add property "Has isolated point"

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

**Comment by StevenClontz** [comment]
PR stems from proposed T307, used at [this answer](https://math.stackexchange.com/questions/4583013/how-many-t-0-topologies-are-possible-for-a-set-of-3-elements-count-only-the-n/4583879#4583879).

Several results on disconnectedness stem from having an isolated point - these could be generalized using proposed P139.

**Comment by prabau** [comment]
This property, or its negation, is going to be very useful. The negation would be "perfect space" (also often used with the totally different meaning of $G_{\delta}$ space) or "dense in itself" (harder for beginner to grasp maybe).  "Has an isolated point" is completely clear and unambiguous.

To help the reviewing, could you add a comment here with the three theorems stated in words?

**Comment by prabau** [comment]
> To help the reviewing, could you add a comment here with the three theorems stated in words?
Never mind, I parsed it.

**Comment by prabau** [comment]
For (S4) (indiscrete two point set), the scattered trait (P51 = false) is being removed.  How is that going to be deduced?

Also, right now the deduction of Empty = false make use of scattered = false.  Also the deduction of Has distinct points = true.  Not saying it cannot be deduced without the scattered trait, but how?

And how will the property Has isolated point = false come about?

File: spaces/S000164/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+uid: S000164
+slug: two-indiscrete-one-isolated
+name: Two-point indiscrete space summed with an isolated point
```
Comment (prabau) [inline spaces/S000164/README.md:4]: Maybe better name: "Two-point indiscrete space summed with singleton".

File: spaces/S000164/properties/P000078.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+space: S000164
+property: P000078
+value: true
+---
+
```
Comment (prabau) [inline spaces/S000164/properties/P000078.md:6]: Looks weird on the browser page if there is no text here and no reference.  Maybe add "By construction" for example.

File: spaces/S000164/properties/P000139.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+space: S000164
+property: P000139
+value: true
+---
+
```
Comment (prabau) [inline spaces/S000164/properties/P000139.md:6]: Same comment.

**Comment by StevenClontz** [comment]
> To help the reviewing, could you add a comment here with the three theorems stated in words?

One of the reasons I've started making PRs by pushing branches to the repo itself is that it triggers @jamesdabbs's GitHub action to make a preview available at <https://topology.pi-base.org/dev>. Change the branch name to the name of the branch I'm PRing and you can browse everything. We need to make *every* PR do this (much less find workflows for folks to preview as they edit, before the PR/PR draft).

**Comment by StevenClontz** [comment]
> For (S4) (indiscrete two point set), the scattered trait (P51 = false) is being removed. How is that going to be deduced?

image


> And how will the property Has isolated point = false come about? Every indiscrete space with at least two points has no isolated point. Pi-base only has one example of such indiscrete space (the one with two points). Still, possibly worth a property, unless it can be deduced from something else.

That one isn't showing - I need to add it. Thanks! I could add the theorem `indiscrete + has distinct points => no isolated point` which would make sure it shows up a few places.

**Comment by prabau** [comment]
> One of the reasons I've started making PRs by pushing branches to the repo itself is that it triggers @jamesdabbs's GitHub action to make a preview available at https://topology.pi-base.org/dev. Change the branch name to the name of the branch I'm PRing and you can browse everything. We need to make _every_ PR do this (much less find workflows for folks to preview as they edit, before the PR/PR draft).

That's pretty cool.  But how do I get to the preview?  https://topology.pi-base.org/dev has some links at the top with Spaces, Properties, Theorems, but these link back to the official website ???

**Comment by prabau** [comment]
I got it now.  Needed to manually enter the branch name in corresponding box.  That is really useful!

**Comment by prabau** [comment]
Let me know if you want me to approve now and leave the extra change for a later PR.

**Review by prabau — APPROVED** [review]

**Pull request merged**